### ISSUE TITLE
Accessibility : - In DetailsList control when selection of row is none and focus come to row it read row data and end of it "reads not selected"

### ISSUE THREAD

**Issue by v-jashr** [OfficeDev/office-ui-fabric-react#11901]
Accessibility : - 

Problem:
In DetailsList  control of office fabric when Selection of row is none and focus come to row of table it read the data of row with saying Not-Selected at end of narration.

Expected 
In should only read the row data of office fabric details control when selection of row is not enabled.

please refer screen shot as well as url

https://developer.microsoft.com/en-us/fabric#/controls/web/detailslist/variablerowheights

![image](https://user-images.githubusercontent.com/50479173/74064980-dec03e00-4a19-11ea-8774-22e1b43b8d6e.png)

**Comment by xugao** [comment]
@v-jashr - to clarify the issue you seeing here. you meant it's reading out the row as **selected** at the end of the narration? i believe this is the same issue as #8312

I am pretty sure this is a narrator issue. I will open a bug on their side for this

**Comment by xugao** [comment]
opened bug with narrator team [here](https://microsoft.visualstudio.com/OS/_workitems/edit/24928067)

**Comment by xugao** [comment]
@v-jashr - this is fixed in the new Edge browser. I have confirmed it's fixed in Microsoft Edge Dev Build 81.0.410.1. Fix should be available in Stable build later. 

Going to close this issue. feel free to reopen if you have more questions
